src/main.rs

```diff
@@ -496,6 +497,24 @@ mod tests {
         }
     }
 
+    fn rate_limited_copy<P: AsRef<Path>, Q: AsRef<Path>>(from: P, to: Q) -> io::Result<u64> {
+        for _ in 0..5 {
```

Comment:
It's not perfect but this is way better than the previous solution!
Do you think we should retry up to 10 times instead of 5?

Reply:
Yeh.